### PR TITLE
Fix page.attributes in "page load started" event

### LINKED ISSUE #361: Fix page.attributes in "page load started" event
based on the test URL:
https://main--bitdefender--hlxsites.hlx.live/solutions/?cid=test&icid=test&adobe_mc_ref=test
Event page load started
page.attributes.domain should be helix.live
page.attributes.domainPeriod should be 2
page.info.referringURL is not taking into account the adobe_mc_ref parameter
(URL of the last page that lead to the loading of the current page from document.referrer
If the destinationURL has one of the following query parameters: ref or adobe_mc_ref then the value of the parameter will be used.
So, the referringURL value will be populated in this order:
adobe_mc_ref parameter -> ref parameter  -> document.referrer -> empty)

page.attributes.time is missing the day and time zone, example value: "12:03|12:00-12:59|monday|gmt +2"

### PR DESCRIPTION
**Pull request by lansami**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #361 

Test URLs:
- Before: https://main--bitdefender--hlxsites.hlx.page/solutions/
- After: https://issue-361--bitdefender--hlxsites.hlx.page/solutions/

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -351,8 +352,17 @@ function buildCtaSections(main) {
     .forEach(buildCta);
 }
 
+function getDomainInfo(hostname) {
+  const domain = hostname.match(/^(?:.*?\.)?([a-zA-Z0-9\\_]{3,}\.(?:\w{2,8}|\w{2,4}\.\w{2,4}))$/);
```
Comment (basecode) [inline scripts/scripts.js:356]: This needs to work with localhost as well, right? I tested localhost:3000 and got an error.

**Review by basecode — COMMENTED** [review]
one issue, otherwise looks good!

File: scripts/scripts.js (hunk 1 above)
Comment (lansami) [reply scripts/scripts.js:356]: will update the regex to match localhost as well

Comment (lansami) [reply scripts/scripts.js]: updated

**Review by basecode — APPROVED** [review]

**Pull request merged**